pkg/local_workflows/whoami_workflow_test.go

```diff
@@ -71,7 +80,7 @@ func Test_WhoAmI_whoAmIWorkflowEntryPoint_happyPath(t *testing.T) {
 
 	mockClient := newTestClient(func(req *http.Request) *http.Response {
 		// Test request parameters
-		assert.Equal(t, "/v1/user/me", req.URL.String())
+		assert.Equal(t, "/rest/self?version=2024-02-26%7Eexperimental", req.URL.String())
```

Comment:
**suggestion:** (non-blocking) Maybe we could introduce a shared constant here to store this value and reduce the number of changes when updating this in the future.